Documentation/ceph-cluster-crd.md

```diff
@@ -1230,6 +1230,7 @@ However, the administrator might want to sanitize the drives in more depth with
   * `dataSource`: indicate where to get random bytes from to write on the disk. Possible choices are 'zero' (default) or 'random'.
   Using random sources will consume entropy from the system and will take much more time then the zero source
   * `iteration`: overwrite N times instead of the default (1). Takes an integer value
+* `uninstallMode`: uninstallMode defines how the uninstall should be performed. Valid values are either an empty string or "yes-really-uninstall-even-if-volumes-in-use".
```

Comment:
Since we're not potentially causing data loss, it's ok to have this as a boolean policy instead of requiring a special flag.
```suggestion
* `allowUninstallWithVolumes`: Whether to uninstall the cluster when volumes still exist. Clients with pods mounted will hang if the cluster is uninstalled. The default is `false`. Set to `true` to force the uninstall.
```

Reply:
Done.